extension/content/personas.css

```diff
@@ -90,6 +95,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
```

Comment:
I think it is not necessary. Because there is no other rule for **toolbar[iconsize="small"]**, it does not get affected from hover or active pseudo rules. 

I tested on SeaMonkey and it works as expected with adding hover and active pseudos.

But let me know if you want to add them.